libbeat/ml-importer/importer.go

```diff
@@ -40,6 +40,9 @@ var (
 	kibanaGetModuleURL    = "/api/ml/modules/get_module/%s"
 	kibanaRecognizeURL    = "/api/ml/modules/recognize/%s"
 	kibanaSetupModuleURL  = "/api/ml/modules/setup/%s"
+
+	// ML assets are not loaded to ES 8.0.0 or newer.
+	incompatibleSince = common.MustNewVersion("8.0.0")
```

Comment:
I know we had in the past problems with comparing our "beta / alpha" naming. Ideally this also don't work with 8.0.0-beta1. But as it is the common package, like this is solved there already.